.github/workflows/e2e-bs-dev.yml

```diff
@@ -0,0 +1,56 @@
+name: e2e browserstack (dev)
+
+on:
+  push:
+    branches:
+      - dev
+      - browserstack-cypress
```

Comment:
We would have to remove the `browserstack-cypress` branch from here before merging the PR, right?

Reply:
Yes. I've just done it.